modules/apps/object/object-test/src/testFunctional/tests/ObjectPartialValidation.testcase

```diff
@@ -21,10 +21,133 @@ definition {
 		}
 	}
 
+	@description = "LPS-182586 - Verify it's possible to see an error message under those fields in the page builder when attempting to add incorrect entries"
+	@priority = 5
+	test CanAddEntryInPageBuilder {
+		property custom.properties = "feature.flag.LPS-187846=true";
+		property portal.acceptance = "true";
+
+		task ("Given a custom object with several fields like a text, integer, decimal and precision decimal") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 192643",
+				objectName = "CustomObject192643",
+				pluralLabelName = "Custom Objects 192643");
+
+			for (var i : list "Text-String,Integer-Integer,Decimal-Double,PrecisionDecimal-BigDecimal") {
+				var fieldBusinessType = StringUtil.extractFirst(${i}, "-");
+				var fieldType = StringUtil.extractLast(${i}, "-");
```

Comment:
I'm doing this since we can't use two lists in one loop for. In this case I need two parameters: fieldBusinessType and fieldType